Author: Andreas Tille <tille@debian.org>
Last-Update: Mon, 06 Jun 2016 11:24:46 +0200
Description: In Debian */*.vschema is installed to /usr/lib/ncbi-vdb/

--- a/test/loaders/fastq-loader/test-fastq-loader.cpp
+++ b/test/loaders/fastq-loader/test-fastq-loader.cpp
@@ -232,7 +232,7 @@ public:
     uint32_t  columnIdx;
 };
 const string TempFileFixture::TempDir = "./tmp";
-const string TempFileFixture::SchemaPath = "align/align.vschema";
+const string TempFileFixture::SchemaPath = "/usr/lib/ncbi-vdb/align/align.vschema";
 const string TempFileFixture::DbType = "NCBI:align:db:alignment_unsorted";
 
 ///////////////////////////////////////////////// FASTQ-based tests for CommonWriter
--- a/tools/loaders/fastq-loader/fastq-loader.c
+++ b/tools/loaders/fastq-loader/fastq-loader.c
@@ -290,7 +290,7 @@ rc_t CC Usage (const Args * args)
 
 /* MARK: Definitions and Globals */
 
-#define SCHEMAFILE "align/align.vschema"
+#define SCHEMAFILE "/usr/lib/ncbi-vdb/align/align.vschema"
 
 CommonWriterSettings G;
 
--- a/tools/loaders/bam-loader/bam-loader.c
+++ b/tools/loaders/bam-loader/bam-loader.c
@@ -596,7 +596,7 @@ rc_t CC Usage (const Args * args)
 
 /* MARK: Definitions and Globals */
 
-#define SCHEMAFILE "align/align.vschema"
+#define SCHEMAFILE "/usr/lib/ncbi-vdb/align/align.vschema"
 
 Globals G;
 
--- a/tools/loaders/sra-sort/run.c
+++ b/tools/loaders/sra-sort/run.c
@@ -78,7 +78,7 @@ bool map_typename_builtin ( const ctx_t
 
 #define ALIGN_EVIDENCE_MAP "NCBI:align:db:alignment_evidence_sorted"
 #define ALIGN_UNSORTED_MAP "NCBI:align:db:alignment_sorted"
-#define ALIGN_SRC          "align/align.vschema"
+#define ALIGN_SRC          "/usr/lib/ncbi-vdb/align/align.vschema"
 
     if ( strcmp ( out, "NCBI/align/db/alignment_evidence" ) == 0 )
         copied = string_copy ( out, size, ALIGN_EVIDENCE_MAP, sizeof ALIGN_EVIDENCE_MAP - 1 );
--- a/tools/loaders/cg-load/cg-load.c
+++ b/tools/loaders/cg-load/cg-load.c
@@ -1074,7 +1074,7 @@ rc_t CC KMain( int argc, char* argv[] )
     const XMLLogger* xml_logger = NULL;
     SParam params;
     memset(&params, 0, sizeof(params));
-    params.schema = "align/align.vschema";
+    params.schema = "/usr/lib/ncbi-vdb/align/align.vschema";
 
     params.argv0 = argv[0];
     
--- a/tools/test-tools/sortreadtest/sortreadtest.c
+++ b/tools/test-tools/sortreadtest/sortreadtest.c
@@ -21,11 +21,11 @@
 
 /*
 
-  default file is "ncbi/seq.vschema"
+  default file is "/usr/lib/ncbi-vdb/ncbi/seq.vschema"
   default table-spec "NCBI:tbl:base_space#2"
 
 */
-#define SCHEMASPEC "ncbi/seq.vschema"
+#define SCHEMASPEC "/usr/lib/ncbi-vdb/ncbi/seq.vschema"
 #define TYPESPEC "NCBI:tbl:base_space#2"
 
 const char UsageDefaultName[] = "sortreadtest";
--- a/tools/internal/vdb-copy/vdb-copy.kfg
+++ b/tools/internal/vdb-copy/vdb-copy.kfg
@@ -25,11 +25,11 @@
 # matching legacy-tables with schema's to write to for vdb-copy
 
 # ILLUMINA
-/VDBCOPY/NCBI_SRA_Illumina_tbl_v0a_1/schema = "sra/illumina.vschema"
+/VDBCOPY/NCBI_SRA_Illumina_tbl_v0a_1/schema = "/usr/lib/ncbi-vdb/sra/illumina.vschema"
 #/VDBCOPY/NCBI_SRA_Illumina_tbl_v0a_1/tab = "NCBI:SRA:Illumina:tbl:q4:v2"
 #/VDBCOPY/NCBI_SRA_Illumina_tbl_v0a_1/tab = "NCBI:SRA:Illumina:tbl:q1:v2"
 /VDBCOPY/NCBI_SRA_Illumina_tbl_v0a_1/tab = "NCBI:SRA:Illumina:tbl:phred:v2"
-/VDBCOPY/_ILLUMINA_/schema = "sra/illumina.vschema"
+/VDBCOPY/_ILLUMINA_/schema = "/usr/lib/ncbi-vdb/sra/illumina.vschema"
 /VDBCOPY/_ILLUMINA_/tab = "NCBI:SRA:Illumina:tbl:phred:v2"
 
 
@@ -41,9 +41,9 @@
 
 
 # ABSOLID
-/VDBCOPY/NCBI_SRA_ABI_tbl_v1_1/schema = "sra/abi.vschema"
+/VDBCOPY/NCBI_SRA_ABI_tbl_v1_1/schema = "/usr/lib/ncbi-vdb/sra/abi.vschema"
 /VDBCOPY/NCBI_SRA_ABI_tbl_v1_1/tab = "NCBI:SRA:ABI:tbl:v2"
-/VDBCOPY/_ABSOLID_/schema = "sra/abi.vschema"
+/VDBCOPY/_ABSOLID_/schema = "/usr/lib/ncbi-vdb/sra/abi.vschema"
 /VDBCOPY/_ABSOLID_/tab = "NCBI:SRA:ABI:tbl:v2"
 
 
--- a/tools/loaders/pacbio-load/pl-context.h
+++ b/tools/loaders/pacbio-load/pl-context.h
@@ -52,7 +52,7 @@ extern "C" {
 #define ALIAS_WITH_PROGRESS "p"
 #define ALIAS_OUTPUT        "o"
 
-#define DFLT_SCHEMA         "sra/pacbio.vschema"
+#define DFLT_SCHEMA         "/usr/lib/ncbi-vdb/sra/pacbio.vschema"
 #define PACBIO_SCHEMA_DB    "NCBI:SRA:PacBio:smrt:db"
 
 
